### PR TITLE
Add QNX/nto support

### PR DESCRIPTION
**Pull request by flba-eb**
cc @gh-tr

### CODE REVIEW

File: src/use_file.rs [hunk 1]
```diff
@@ -22,7 +22,7 @@ use core::{
 // documentation indicates that /dev/urandom is insecure.
 // On Solaris/Illumos, see src/solaris_illumos.rs
 // On Dragonfly, Haiku, and macOS, the devices are identical.
-#[cfg(any(target_os = "solaris", target_os = "illumos"))]
+#[cfg(any(target_os = "solaris", target_os = "illumos", target_os = "nto"))]
```
Comment (gh-tr) [inline src/use_file.rs]: Should be using /dev/urandom, not /dev/random (please see the patches I sent you).  Just enable below.

File: src/util_libc.rs [hunk 2]
```diff
@@ -38,6 +38,8 @@ cfg_if! {
 cfg_if! {
     if #[cfg(target_os = "vxworks")] {
         use libc::errnoGet as get_errno;
+    } else if #[cfg(target_os = "nto")] {
```
Comment (gh-tr) [inline src/util_libc.rs]: errno, errno_location, ___errno, etc. all all C functions for other *nix distros.  Any reason you didn't implement this like all other distro's did in the block above?

**Review by gh-tr — CHANGES_REQUESTED** [review]

File: src/use_file.rs (hunk 1 above)
Comment (flba-eb) [reply src/use_file.rs]: Fixed

File: src/util_libc.rs (hunk 2 above)
Comment (flba-eb) [reply src/util_libc.rs]: Fixed.

Comment (flba-eb) [reply src/util_libc.rs]: Thanks!

**Review by gh-tr — APPROVED** [review]

**Pull request closed without merging**